### PR TITLE
Backend service

### PR DESCRIPTION
**Pull request by Zenmai0822**
This PR contains the following changes:

- Added a singleton service that handles all the requests made to the API backend. 

The service takes in `json`s unless the API uses path variables. I will change them if func parameters are more desired.

### CODE REVIEW

File: src/services/backendService.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
+export default class BackendService { 
+
+    static urlPrefix = "http://http://www.doublewb.xyz/hci"
```
Comment (l-tn3314) [inline src/services/backendService.js]: have an extra `http://`

File: src/services/backendService.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            header: {"content-type": "application/json"}
+        }).then(response => { 
+            response.clone().json();
+        })
+    }
+
+    async removeParticipation(item) { 
+        return fetch(BackendService.urlPrefix + "/amounts", {
+            method: "delete", 
+            body: JSON.stringify(item);
```
Comment (l-tn3314) [inline src/services/backendService.js]: comma

**Comment by Zenmai0822** [comment]
I guess I shouldn't code after an interview -- sorry for the typos! 😅

File: src/services/backendService.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
+export default class BackendService { 
+
+    static urlPrefix = "http://www.doublewb.xyz/hci"
```
Comment (l-tn3314) [inline src/services/backendService.js]: before merging, we should change this to `https`

**Pull request merged**